build/NuSpecs/ProjectReunionFrameworkPackage.nuspec

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>$ID$</id>
+    <version>0.0.0-SpecifyVersionOnCommandline</version>
+    <title>ProjectReunion</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <description>Project Reunion</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>Windows Reunion ProjectReunion</tags>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="file">license.txt</license>
+    <projectUrl>https://github.com/microsoft/projectreunion</projectUrl>
+    <iconUrl>https://aka.ms/winui_icon</iconUrl>
+  </metadata>
+  <files>
+    <file src="readme.txt"/>
+    <file src="license.txt"/>
+
+<!--    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\sdk\Microsoft.ProjectReunion.winmd"/> -->
+    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd"/>
+    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion.FrameworkPackagePRI\Microsoft.ProjectReunion.pri"/>
+
```

Comment:
I don't think we can't use the pri file directly from the framework package because the framework package one will have the app name specified as the framework package but the one we put in the nuget package needs to be merged into the consume app's resources. I could be wrong so if you've tried it and it works then ... cool. This would be something we could simplify in MUX then.

Reply:
Framework packages can have resources.pri. It's used for localizable manifested values (DisplayName, etc) plus if anything in the framework needs localization (probably not much but non-zero). Framework packages can't have associated Resource packages (unlike Main+Optional that can). Thus any localization values need to be self-contained in the Framework's resources.pri (and image files)

The framework won't have much that's localizable but DisplayName, PublisherDisplayName and Description (if any) should be. Logo can be, though unclear if we need to localize it. Probably not localization per se (language, home region, etc) but for scale 100 vs 150 vs 200 etc.

Wouldn't hurt to change DisplayName etc from e.g. "Microsoft Project Reunion" to "ms-resource:DisplayName" and stick the en-us value in resources.pri, to ease later localization.

Reply:
EHO copied all this from the microsoft-ui-xaml repo, I'm quite familiar with it as I wrote most of it to begin with. :) What I was calling out is that the resources.pri that's included in the framework package and the resources.pri that is included in the nuget package cannot be binary identical ... or at least I thought they couldn't be because I thought they needed different IndexName for the ResourceMap. But now that I look into it, maybe they can be the same -- which would be great, that would simplify things in WinuI if we could.

Reply:
@axelandrejs ?

Reply:
Short answer: You should be able to use the same resources in both as we will move the namespace for you.

Long answer.

The requirement for any packaged resource is that the root namespace must match the package family name. So if you have the foo app and the bar framework package, the resources would be ms-resource://foo/... and ms-resource://bar/.., respectively.

If you include a NuGet-based library project into your app, it will produce an interim PRI file (in this case Microsoft.ProjectReunion.pri). Then when the app is assembled this interim file is merged into the main resources.pri alongside other libraries as well as the main app resources.

As part of this process, by default the pipeline moves the resource into the app namespace.  EG, for a WinUI app.

Interim file:

ms-resource://Microsoft.UI.Xaml/Files/Microsoft.UI.Xaml/DensityStyles/Compact.xaml

Final file:

ms-resource://2dbd2f29-2d2b-49ea-a8a2-27434fd83db0/Files/Microsoft.UI.Xaml/DensityStyles/Compact.xaml

This means that there should be no conflict between the two.

Reply:
1) I haven't tested any of the framework package items, this is step 3 (checking in step 1).  But it was too hard to dis-entangle files so I brought over almost everything in step 1. 
2) What is the suggested change here?  I can make the change and then test it in step 3 (and step 2, Reunion metapackage might have to happen after we publish to the nuget feed since I think package references in nuget have to reference something publicly recachable.

Reply:
No change for reunion, but it's something we could look into simplifying for MUX.